cmd/buildah/import.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/containers/storage/pkg/archive"
+	"github.com/pkg/errors"
+	"github.com/projectatomic/buildah"
+	"github.com/urfave/cli"
+)
+
+var (
+	importFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "change, c",
+			Usage: "Apply specified Dockerfile instructions while importing the image",
+		},
+		cli.StringFlag{
+			Name:  "message, m",
+			Usage: "Set commit message for imported image",
+		},
+	}
+	importCommand = cli.Command{
+		Name:        "import",
+		Usage:       "Create an empty filesystem image and import the contents of the tarball.",
+		Description: `Create an empty filesystem image and import the contents of the tarball (.tar, .tar.gz, .tgz, .bzip, .tar.xz, .txz) into it, then optionally tag it.`,
+		Flags:       importFlags,
+		Action:      importCmd,
+		ArgsUsage:   "CONTAINER",
+	}
+)
+
+func importCmd(c *cli.Context) error {
+	var (
+		tag string
+		err error
+	)
+
+	args := c.Args()
+	if len(args) == 0 {
+		return errors.Errorf("tar file must be specified")
+	}
+	tarFile := args[0]
+
+	if len(args) == 2 {
+		tag = args[1]
+	}
+	if len(args) > 2 {
+		return errors.Errorf("too many arguments specified")
+	}
+
+	change := c.String("change")
+	fmt.Println("Change=", change)
+	message := c.String("message")
+	fmt.Println("Message=", message)
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+
+	options := buildah.BuilderOptions{
+		FromImage: "scratch",
+		Container: tag,
+	}
+
+	builder, err := buildah.NewBuilder(store, options)
+	if err != nil {
+		return err
+	}
```

Comment:
If the container is intended to be temporary, you probably want to defer a call to its `Delete` method here.